worksheet.md

```diff
@@ -131,6 +131,6 @@ Some of the cool features on the micro:bit are the GPIO pins, the accelerometer,
 - As always, the best way of finding out how to use a new piece of hardware or software (in this case both) is to have a look at the documentation. You can find documentation for MicroPython on the micro:bit at [this site](https://microbit-micropython.readthedocs.org/en/latest/).
 - You could have a go at some of the other cool projects using a Raspberry Pi and a micro:bit. Have a look at the following resources:
 1. [micro:bit selfies](https://www.raspberrypi.org/learning/microbit-selfies)
-2. [micro:bit game controller](https://www.raspberrypi.org/learning/microbit-game-controller)
-2. [micro:bit meteorologist](https://www.raspberrypi.org/learning/microbit-meteorologist)
+2. [micro:bit interactive-badge](https://projects.raspberrypi.org/en/projects/interactive-badge)
```

Comment:
Should just be `interactive badge` without the dash, since that's just what displays on the page

Reply:
2nd url possibly can be `https://projects.raspberrypi.org/en/projects/microbit-meteorologist` as that's the end of the redirect

Reply:
You right, I missed that. Fixed! Thank you